src/test/java/kitchenpos/bo/OrderBoTest.java

```diff
@@ -0,0 +1,219 @@
+        orderLineItem.setMenuId(1l);
+
+        final OrderLineItem orderLineItem1 = new OrderLineItem();
+        orderLineItem1.setMenuId(2l);
+        order.setOrderTableId(1l);
+        order.setOrderLineItems(Arrays.asList(orderLineItem, orderLineItem1));
+
+        final OrderTable orderTable = Fixtures.orderTable;
+        orderTable.setEmpty(true);
+
+        given(menuDao.countByIdIn(Arrays.asList(orderLineItem.getMenuId(),
+                orderLineItem1.getMenuId()))).willReturn(2l);
+        given(orderTableDao.findById(order.getOrderTableId())).willReturn(Optional.of(orderTable));
+
+        assertThatExceptionOfType(IllegalArgumentException.class)
+                .isThrownBy(() -> orderBo.create(order));
+    }
+
+    @Test
+    @DisplayName("주문 테이블이 정상 등록")
+    void create_success() {
+        final Order order = new Order();
+        order.setOrderTableId(1l);
+
+        final OrderLineItem orderLineItem = new OrderLineItem();
+        orderLineItem.setMenuId(1l);
+
+        final OrderLineItem orderLineItem1 = new OrderLineItem();
+        orderLineItem1.setMenuId(2l);
+
+        order.setOrderLineItems(Arrays.asList(orderLineItem, orderLineItem1));
+
+        final OrderTable orderTable = Fixtures.orderTable;
+        orderTable.setEmpty(false);
+
+        given(menuDao.countByIdIn(Arrays.asList(orderLineItem.getMenuId(),
+                orderLineItem1.getMenuId()))).willReturn(2l);
+        given(orderTableDao.findById(order.getOrderTableId())).willReturn(Optional.of(orderTable));
+        given(orderDao.save(order)).willReturn(order);
+        given(orderLineItemDao.save(orderLineItem)).willReturn(orderLineItem);
+        given(orderLineItemDao.save(orderLineItem1)).willReturn(orderLineItem1);
+
+        final Order result = orderBo.create(order);
+
+        assertThat(result).isNotNull();
+    }
+
+    @Test
+    @DisplayName("주문 목록을 조회 할 수 있다.")
+    void list() {
+        final Order order = new Order();
+        order.setId(1l);
+
+        final OrderLineItem orderLineItem = new OrderLineItem();
+        orderLineItem.setOrderId(1l);
+
+        final OrderLineItem orderLineItem1 = new OrderLineItem();
+        orderLineItem1.setOrderId(1l);
+
+        final List<OrderLineItem> orderLineItems = Arrays.asList(orderLineItem, orderLineItem1);
+        order.setOrderLineItems(orderLineItems);
+
+        given(orderDao.findAll()).willReturn(Arrays.asList(order));
+
+        given(orderLineItemDao.findAllByOrderId(order.getId())).willReturn(orderLineItems);
+
+        final List<Order> list = orderBo.list();
+
+        assertThat(list).isNotEmpty();
+        assertThat(list.get(0).getOrderLineItems().size()).isEqualTo(orderLineItems.size());
+    }
+
+    @Test
+    @DisplayName("주문이 존재 하지 않는 경우")
+    void changeOrderStatus_notExist_order() {
+        final Order order = new Order();
+        order.setId(1l);
+
+        given(orderDao.findById(order.getId())).willThrow(IllegalArgumentException.class);
```

Comment:
아래의 코드로 개선해 보면 어떨까요?

```suggestion
        given(orderDao.findById(order.getId())).willReturn(Optional.empty());
```

Reply:
수정했습니다.